server/routes/user.js

```diff
@@ -0,0 +1,27 @@
+/* eslint new-cap: "off" */
+const express = require('express');
+
+const router = express.Router();
+const User = require('../controllers/user');
+const Authentic = require('../controllers/middleware');
+
+router.route('/users/login')
+  .post(User.login);
+
+router.route('/users')
+  .post(User.create);
+
+router.use(Authentic.Auth);
+
+router.route('/users')
+  .get(User.all);
+
+router.route('/users/:id')
+  .get(User.get)
+  .delete(User.delete)
+  .put(User.update);
+
+router.route('/users/logout')
```

Comment:
**Code Quality - Critical**

Routes have an Order of Operation.
`users/logout` matches `/users/:id` hence `users/logout` will not get called because a matching route, precedes it.

You could use `user/login` and `user/logout` for your actions, so they differ from your resources, or place them before the multi-matching route declarations.